tauri-api/Cargo.toml

```diff
@@ -35,6 +35,7 @@ tauri-utils = { version = "0.5", path = "../tauri-utils" }
 clap = { version = "=3.0.0-beta.2", optional = true }
 notify-rust = { version = "4.2.2", optional = true }
 once_cell = "1.5.2"
+tauri-hotkey = { git = "https://github.com/tauri-apps/tauri-hotkey-rs", branch = "dev", optional = true }
```

Comment:
Shouldn't this get published to crates.io before we release? Do we want to do it before merge so we don't forget?

Reply:
we will publish it, but it's still in development so i'll just leave it as a git dep here. (yeah let's hope we don't forget it :joy: )

Reply:
We might want to consider making a list or adding it to that hackmd at least.

Reply:
yeah we should manage our TODOs better